rollup.config.js

```diff
@@ -0,0 +1,20 @@
+import path from 'path';
+import inject from 'rollup-plugin-inject';
+import legacy from 'rollup-plugin-legacy';
+
+export default {
+  input: 'src/L.GridLayer.MaskCanvas.js',
+  output: {
+    file: 'dist/L.GridLayer.MaskCanvas.js',
+    format: 'iife'
```

Comment:
not really necessary here because you source code is written in ES5 and extends the global variable `L`. You'd need the `extend` if your source code uses ES6 modules or AMD, also when you output to UMD.
In this case, because the code does not export anything there's no need to use that option